ca_on_toronto/events-incremental.py

```diff
@@ -203,7 +203,12 @@ def is_being_introduced(item, event):
 
     def agenda_from_url(self, url):
         page = self.lxmlize(url)
-        main = page.xpath('//table[1]/..')[0]
+        try:
+            main = page.xpath('//table[1]/..')[0]
+        except IndexError:
```

Comment:
Can you add a comment (and include a sample URL) explaining why the xpath would sometimes fail?

Also, `page.xpath('//table[1]/..')` will be falsy if nothing is found, so instead of immediately indexing `[0]`, I recommend checking the truthiness of `page.xpath('//table[1]/..')`, so:

```python
main = page.xpath('//table[1]/..')
if main:
  main = main[0]
else:
  return []